### PR TITLE
Update contributing guide (added a new feature while doing so)

### PR DESCRIPTION
**Pull request by jmaslek**
I added a new function so that I could live through reading the documentation

### CODE REVIEW

**Comment by jmaslek** [comment]
I also added codecov because it took 1 line.

File: openbb_terminal/stocks/fundamental_analysis/fa_controller.py [hunk 1]
```diff
@@ -1821,17 +1805,27 @@ def call_pt(self, other_args: List[str]):
...
+                    data=self.stock,
+                    start_date=self.start,
+                    limit=ns_parser.limit,
+                    raw=ns_parser.raw,
+                    export=ns_parser.export,
+                    sheet_name=" ".join(ns_parser.sheet_name)
+                    if ns_parser.sheet_name
+                    else None,
+                )
+            elif ns_parser.source == "FinancialModelingPrep":
```
Comment (hjoaquim) [inline openbb_terminal/stocks/fundamental_analysis/fa_controller.py:1820]: Wasn't able to test it (special endpoint) - I've just changed a little the message to provide more context.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -81,42 +100,64 @@ will be implemented.
...
+- We check for different error messages.  This will depend on the API provider and usually requires some trial and error.  With the FMP api, if there is an invalid symbol, we get a response code of 200, but the json response has an error message field.  Same with an invalid api key.
+- When an error is caught, we still return an empty dataframe.
+- We return the json response as a pandas dataframe.  Most functions in the terminal should return a datatframe, but if not, make sure that the return type is specified.
+
 Note:
 
-1. As explained before, it is possible that this file needs to be created under `common/` directory rather than
-   `stocks/`, which means that when that happens this function should be done in a generic way, i.e. not mentioning stocks
-   or a specific context.
+1. As explained before, it is possible that this file needs to be created under `common/` directory rather than `stocks/`, which means that when that happens this function should be done in a generic way, i.e. not mentioning stocks or a specific context.
```
[the remark this answers is not in the document]
Comment (hjoaquim) [reply CONTRIBUTING.md:163]: tested now. works like a charm:
image

File: openbb_terminal/stocks/fundamental_analysis/fa_controller.py (hunk 1 above)
Comment (hjoaquim) [reply openbb_terminal/stocks/fundamental_analysis/fa_controller.py:1820]: tested now. works like a charm:
image

**Review by hjoaquim — APPROVED** [review]
Looks great IMO, I think this PR touched a lot of stuff, and our contributing guidelines are now completely up-to-date!

**Pull request merged**